hub-config/admin.json

```diff
@@ -8,5 +8,13 @@
     },
     "repository_url": "https://github.com/reichlab/variant-nowcast-hub",
     "file_format": ["parquet"],
-    "timezone": "US/Eastern"
+    "timezone": "US/Eastern",
+    "cloud": {
+        "enabled": false,
```

Comment:
confirming that we want the cloud set-up here but we have the enabled switch purposely at "false" for now so nothing gets sycned yet.

Reply:
Yep--wanted to signal our intent to have this on the cloud, but we haven't yet provisioned any of the infrastructure (and I didn't want to make assumptions about what the name of the S3 bucket should be).